commit 9949bb054e7a
Author: Greg V <greg@unrelenting.technology>
Date:   Mon Feb 24 20:18:01 2020 +0000

    Bug 1594027 - add more STL wrappers to fix build with libc++ 9 r=froydnj
    
    Differential Revision: https://phabricator.services.mozilla.com/D62847
    
    --HG--
    extra : moz-landing-system : lando
---
 config/gcc-stl-wrapper.template.h | 8 ++++++++
 1 file changed, 8 insertions(+)

Taken from www/firefox

diff --git config/gcc-stl-wrapper.template.h config/gcc-stl-wrapper.template.h
index 41ae14024b0c0..9fe70fbaab8dc 100644
--- mozilla-release/config/gcc-stl-wrapper.template.h
+++ mozilla-release/config/gcc-stl-wrapper.template.h
@@ -32,6 +32,14 @@
 #  define moz_dont_include_mozalloc_for_cstdlib
 #endif
 
+#ifndef moz_dont_include_mozalloc_for_type_traits
+#  define moz_dont_include_mozalloc_for_type_traits
+#endif
+
+#ifndef moz_dont_include_mozalloc_for_limits
+#  define moz_dont_include_mozalloc_for_limits
+#endif
+
 // Include mozalloc after the STL header and all other headers it includes
 // have been preprocessed.
 #if !defined(MOZ_INCLUDE_MOZALLOC_H) && \
